src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js

```diff
@@ -0,0 +1,415 @@
+                Our mail order pharmacy will send your prescriptions to the
+                address we have on file for you. We ship to all addresses in the
+                United States and its territories. We don’t ship prescriptions
+                to foreign countries.
+              </p>
+              <p>
+                <strong>Important note:</strong> If you change your address
+                within My HealtheVet, it does <strong>not</strong> change your
+                address for prescription shipments. Please contact the VA health
+                facility where you get care to have them update your address on
+                file.
+                <br />
+                <a href="/find-locations/">Find your VA health facility</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        itemScope
+        itemType="http://schema.org/Question"
+        data-template="paragraphs/q_a"
+        data-entity-id={3268}
+        data-analytics-faq-section
+        data-analytics-faq-text="How long will my prescriptions take to arrive, and when should I reorder?"
+      >
+        <h2 itemProp="name" id="how-long-will-my-prescriptions">
+          How long will my prescriptions take to arrive, and when should I
+          reorder?
+        </h2>
+        <div
+          data-entity-id={3268}
+          itemProp="acceptedAnswer"
+          itemScope
+          itemType="http://schema.org/Answer"
+        >
+          <div itemProp="text">
+            <div
+              data-template="paragraphs/wysiwyg"
+              data-entity-id={3269}
+              className="processed-content"
+            >
+              <p>
+                Prescriptions usually arrive within 3 to 5 days. But you’ll be
+                able to find specific information about your order on the
+                website of the delivery service shown in your Rx Tracker.
+              </p>
+              <p>
+                To make sure you have your medicine in time, you’ll want to
+                request your refill at least 10 days before you’ll run out of
+                your current prescription.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        itemScope
+        itemType="http://schema.org/Question"
+        data-template="paragraphs/q_a"
+        data-entity-id={3270}
+        data-analytics-faq-section
+        data-analytics-faq-text="Will my personal health information be protected?"
+      >
+        <h2 itemProp="name" id="will-my-personal-health-inform">
+          Will my personal health information be protected?
+        </h2>
+        <div
+          data-entity-id={3270}
+          itemProp="acceptedAnswer"
+          itemScope
+          itemType="http://schema.org/Answer"
+        >
+          <div itemProp="text">
+            <div
+              data-template="paragraphs/wysiwyg"
+              data-entity-id={3271}
```

Comment:
I think you can sanitize this a bit by removing the `data-` attributes since it's React-based now

Reply:
Good idea! I will do this for all the PRs

Reply:
Done! 🎉